tools/ci_build/github/windows/setup_env_gpu.bat

```diff
@@ -0,0 +1,2 @@
+set PATH=C:\azcopy;C:\local\TensorRT-8.0.1.6.Windows10.x86_64.cuda-11.3.cudnn8.2\lib;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.1\bin;C:\local\cudnn-11.4-windows-x64-v8.2.2.26\cuda\bin;%PATH%
```

Comment:
We have 3 different CUDA version here: 11.1, 11.3 and 11.4

Reply:
Changed to use CUDA11.4
But for TensorRT, it seems only has libraries built against CUDA11.3

Reply:
yes, TRT 8.0 was built against CUDA11.3 according to the name ,but it is compatible with CUDA 11.4

Reply:
You don't need CUDNN being here.